### ISSUE TITLE
Display used space in main navigation bar

### ISSUE THREAD

**Issue by gibberfishinc** [gibberfishinc/gibberfish-nextcloud#1]
Place a widget showing the amount of quota space used on the main nav bar, above the "Deleted Files" link. This should be similar to the one found on the users' profile page.

![space](https://cloud.githubusercontent.com/assets/26328535/24613223/ad5a3c60-1855-11e7-82df-72c500223054.png)
![profilespace](https://cloud.githubusercontent.com/assets/26328535/24613224/ad60c17a-1855-11e7-8a8e-fedef6e1f72b.png)

**Comment by rebecca-grenier-rp** [comment]
I question the idea of putting an informative label in the navigation bar.  Everything else in that list to the right is a link to navigate somewhere.  If you want that information on this page it should be at the top, like on the profile page, so people know it is not somewhere you can go.

**Comment by gibberfishinc** [comment]
Fair enough. We consulted a UX researcher and that was the spot he suggested, but I think his main concern is that it's clearly visible somewhere without having to drill down through menus. We're open to suggestions.

Thanks :)
